core/src/banking_stage.rs

```diff
@@ -415,6 +412,19 @@ impl BankingStage {
         Ok(())
     }
 
+    fn log_tx_errors(result: &transaction::Result<()>) {
+        match result {
+            Err(TransactionError::InstructionError(index, err)) => {
+                debug!("instruction error {:?}, {:?}", index, err);
```

Comment:
We're already posting some metrics with Bank errors elsewhere. Any way to tie into that?